Jellyfin.Api/Controllers/PlaystateController.cs

```diff
@@ -73,15 +74,21 @@ public class PlaystateController : BaseJellyfinApiController
         public ActionResult<UserItemDataDto> MarkPlayedItem(
             [FromRoute] Guid userId,
             [FromRoute] Guid itemId,
-            [FromQuery] DateTime? datePlayed)
+            [FromQuery] string? datePlayed)
         {
+            DateTime? parsedDatePlayed = null;
+            if (!string.IsNullOrEmpty(datePlayed))
+            {
+                parsedDatePlayed = DateTime.ParseExact(datePlayed, "yyyyMMddHHmmss", CultureInfo.InvariantCulture, DateTimeStyles.AssumeUniversal);
```

Comment:
Would it make sense to do a Type Converter instead? https://stackoverflow.com/a/52822295

Reply:
learn something new every day!